packages/cli/src/commands/deploy/index.ts

```diff
@@ -82,6 +82,7 @@ export default async (client: Client) => {
       '--prebuilt': Boolean,
       '--prod': Boolean,
       '--confirm': Boolean,
+      '--cwd': String,
```

Comment:
Can we make this a global flag so it works with other subcommands?

I think we can simplify this to be `process.chdir(cwd)` before any subcommands run.

Reply:
That sounds good to me. Updated.